suite-native/app/app.config.ts

```diff
@@ -4,28 +4,22 @@ import { ExpoConfig, ConfigContext } from 'expo/config';
 
 import { suiteNativeVersion } from './package.json';
 
-type BuildType = 'debug' | 'develop' | 'staging' | 'production';
+type BuildType = 'debug' | 'develop' | 'production';
 
-const getBuildType = (): BuildType => {
-    if (process.env.ENVIRONMENT === 'production') return 'production';
-    if (process.env.ENVIRONMENT === 'staging') return 'staging';
-    if (process.env.ENVIRONMENT === 'develop') return 'develop';
-    return 'debug';
-};
-const buildType = getBuildType();
+const environment = process.env.ENVIRONMENT as BuildType;
+
+const buildType = environment ?? ('debug' as BuildType);
```

Comment:
I believe type cast for 'debug' string is redundant here
```suggestion
const buildType = environment ?? 'debug';
```

Reply:
https://github.com/trezor/trezor-suite/pull/10898/commits/3a9f50c039c08a67c28fe25c017ef8384cfd4165